azure-pipelines.yml

```diff
@@ -57,19 +57,19 @@ jobs:
           setvar skip-github TRUE
         fi
       displayName: 'Create and Push Release Tag'
-    - task: DownloadPipelineArtifact@0
+    - task: DownloadPipelineArtifact@2
```

Comment:
what is the `@2` suffix naming here?

Reply:
https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/download-pipeline-artifact-v0?view=azure-pipelines is deprecated and had to be updated to https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/download-pipeline-artifact-v2?view=azure-pipelines

I don't think it's strictly necessary in this specific case since we only use functionality available in v0, but worth doing now since v0 is deprecated.